.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,54 @@
+<gradleEnterprise>
+  <server>
+    <url>https://ge.apache.org</url>
+    <allowUntrusted>false</allowUntrusted>
+    <!--
+    credentials provided by jenkins exposing GRADLE_ENTERPRISE_ACCESS_KEY
+    apache members can create their own on the server
+    -->
+  </server>
+  <!-- options are documented at https://docs.gradle.com/enterprise/maven-extension/#configuring_background_uploading -->
+  <buildScan>
+    <backgroundBuildScanUpload>#{isFalse(env['CI'])}</backgroundBuildScanUpload>
```

Comment:
From what I see, your CI builds run in Jenkins. From my knowledge, Jenkins does not set a `CI` environment variable. We recommend looking for `JENKINS_URL` instead for these checks in Jenkins.

Reply:
we do it ourselves :D https://github.com/apache/james-project/blob/master/Jenkinsfile#L39